### PR TITLE
More API endpoints

### PR DESCRIPTION
**Pull request by rathorevaibhav**
Targets #4 

### Summary
1. CRUD endpoints for:
    1. Student
    2. Teacher
    3. School
    4. Enrollment Record
2. Updated postman collection as well with these new endpoints.

**DB Schema link**
https://whimsical.com/af-db-schema-8MT1bQVfQx4SqffSYPKiKm

### CODE REVIEW

**Review by pritamps — APPROVED** [review]
Looks good to me. But get approval from @sudheshna-donthineni before merging.

**Review by sudheshna-donthineni — APPROVED** [review]

**Pull request merged**
